src/components/accordion/accordion.e2e.ts

```diff
@@ -47,23 +47,58 @@ describe("calcite-accordion", () => {
-    <calcite-accordion-item heading="Accordion Title 1" icon-start="car" id="1">Accordion Item Content
-    </calcite-accordion-item>
-    <calcite-accordion-item heading="Accordion Title 1" id="2" expanded>Accordion Item Content
-    </calcite-accordion-item>
-    <calcite-accordion-item heading="Accordion Title 3" icon-start="car" id="3">Accordion Item Content
-    </calcite-accordion-item>
-    </calcite-accordion>`);
-    const icon1 = await page.find(`calcite-accordion-item[id='1'] >>> .${CSS.iconStart}`);
-    const icon2 = await page.find(`calcite-accordion-item[id='2'] >>> .${CSS.iconStart}`);
-    const icon3 = await page.find(`calcite-accordion-item[id='3'] >>> .${CSS.iconStart}`);
-    expect(icon1).not.toBe(null);
-    expect(icon2).toBe(null);
-    expect(icon3).not.toBe(null);
+  describe("icon behavior", () => {
+    let page: E2EPage;
+    const scale = { l: "l", m: "m", s: "s" };
+    const accordionItemContent = html`<calcite-accordion-item icon-start="car" id="1"></calcite-accordion-item>
+      <calcite-accordion-item id="2"></calcite-accordion-item>
+      <calcite-accordion-item icon-start="car" id="3"></calcite-accordion-item>`;
+
+    beforeEach(async () => {
+      page = await newE2EPage();
+      await page.setContent(html`<calcite-accordion scale="l"> ${accordionItemContent} </calcite-accordion>`);
+      await page.waitForChanges();
+    });
+
+    it("renders icon if requested", async () => {
+      const icon1: E2EElement = await page.find(`calcite-accordion-item[id='1'] >>> .${CSS.iconStart}`);
+      const icon2: E2EElement = await page.find(`calcite-accordion-item[id='2'] >>> .${CSS.iconStart}`);
+      const icon3: E2EElement = await page.find(`calcite-accordion-item[id='3'] >>> .${CSS.iconStart}`);
+      expect(icon1).not.toBe(null);
+      expect(icon2).toBe(null);
+      expect(icon3).not.toBe(null);
+    });
+
+    it("renders m scale icon for l scale accordion-item", async () => {
+      const item1: E2EElement = await page.find(`calcite-accordion-item[id='1']`);
+      expect(await item1.getProperty("scale")).toBe(scale.l);
+      const icon1: E2EElement = await page.find(`calcite-accordion-item[id='1'] >>> .${CSS.iconStart}`);
+      expect(await icon1.getProperty("scale")).toBe(scale.m);
+    });
+
+    it("renders corresponding scale on accordion-item when parent scale changes, icon scale not affected", async () => {
+      const accordion: E2EElement = await page.find("calcite-accordion");
+      await accordion.setProperty("scale", scale.s);
+      await page.waitForChanges();
+
+      const item1: E2EElement = await page.find(`calcite-accordion-item[id='1']`);
+      expect(await item1.getProperty("scale")).toEqual(scale.s);
+
+      const icon1: E2EElement = await page.find(`calcite-accordion-item[id='1'] >>> .${CSS.iconStart}`);
+      expect(await icon1.getProperty("scale")).toEqual(scale.m);
```

Comment:
Should this icon be a scale `s`?

Reply:
> Erik also brought up that we should not override an item with the parent if a user sets it individually to a different scale, although I'm not sure what the use case for that would be.

Not sure what the specific use case would be either, but I still think that if the `scale` property is explicitly applied either as an attribute or by setting the property directly on the child component, that this value should be honored and not overridden by the parent component.  We can actively discourage doing this in the documentation, and encourage users to set the value at the parent level to ensure visual consistency, but I don't think we should break from convention with regards to how property values are honored as the individual component level.